src/runtime/wasm.ts

```diff
@@ -526,6 +527,15 @@ export class WasmParticle extends Particle {
     }
     this.innerParticle = this.exports[fn]();
     this.container.register(this, this.innerParticle);
+    // TODO(sjmiles): probably too soon; we need to render at least once, but we may have handle
+    // work pending ... consult with Shane
+    setTimeout(() => this.renderOutput(), 100);
```

Comment:
I don't think this is right. First, I would think particle implementations should decide when to render, not this class. Second, a timed delay is always going to be brittle. If you want a particle to render as soon as it can, that would usually mean once all of its readable handles have been synced, and for C++ I have added a mechanism to just that (call `autoRender` in the constructor).